utils/sub8_msgs/msg/Waypoint.msg

```diff
@@ -0,0 +1,4 @@
+# Waypoint
+
+geometry_msgs/Pose pos
```

Comment:
Convention is to use pose and twist as the names of the members, see: [odometry msg definition](http://docs.ros.org/jade/api/nav_msgs/html/msg/Odometry.html)

i.e. 

    geometry_msgs/Pose pose
    geometry_msgs/Twist twist